src/big/big_decimal.cr

```diff
@@ -238,16 +267,21 @@ struct BigDecimal < Number
     self
   end
 
-  # Converts to integer. Truncates anything on the right side of the decimal point.
-  def to_i
+  # Converts to `Int64`. Truncates anything on the right side of the decimal point.
+  def to_i64
     if @value >= 0
-      (@value / TEN ** @scale).to_i
+      (@value / TEN ** @scale).to_i64
     else
-      -(@value.abs / TEN ** @scale).to_i
+      -(@value.abs / TEN ** @scale).to_i64
     end
   end
 
-  # Converts to unsigned integer. Truncates anything on the right side of the decimal point,
+  # Converts to `Int32`. Truncates anything on the right side of the decimal point.
+  def to_i
```

Comment:
If we're doing this why not add the full complement of `to_` methods?

Reply:
Will do.